integration/configs.go

```diff
@@ -27,7 +29,24 @@ const (
 	caCertFile             = "certs/root.crt"
 	serverCertFile         = "certs/server.crt"
 	serverKeyFile          = "certs/server.key"
-	storeConfigTemplate    = `
+)
+
+// Generate a 6 character (a-z,0-9) id for the test being run.
+// This ID is used to generate a network name for the test
+func genUniqueTestID() string {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
This is (posibly re-)seeding global RNG. In general I'd avoid doing that, and prefer to use local instance instead. But for integration tests, that should be fine.